etc/renderd.conf

```diff
@@ -0,0 +1,16 @@
+[renderd]
+num_threads=12
+tile_dir=/var/lib/mod_tile
+stats_file=/var/run/renderd/renderd.stats
```

Comment:
Don't we also need `socketname=/var/run/renderd/renderd.sock` line?

Reply:
It’s commented out in the [default version](https://github.com/openstreetmap/mod_tile/blob/master/renderd.conf), so I suspect that’s the default path and thus doesn’t need to be declared explicitly. I was able to load tiles with a container built from this revision.